op-deployer/pkg/deployer/pipeline/preinstall_l1_dev_genesis.go

```diff
@@ -0,0 +1,20 @@
+package pipeline
+
+import (
+	"fmt"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/interopgen/deployers"
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/state"
+)
+
+func PreinstallL1DevGenesis(env *Env, intent *state.Intent, st *state.State) error {
+	lgr := env.Logger.New("stage", "preinstall-l1-dev-genesis")
+	lgr.Info("Adding preinstalls to L1 dev genesis")
+
+	if err := deployers.InsertPreinstalls(env.L1ScriptHost); err != nil {
```

Comment:
[nit] is there any way we could move this `deployers` package out of interopgen? seems strange for op-deployer to have to depend on interopgen.